corehq/apps/app_manager/views.py

```diff
@@ -961,6 +962,13 @@ def view_generic(req, domain, app_id=None, module_id=None, form_id=None, is_user
         context.update({
             'case_properties': get_all_case_properties(app),
         })
+
+        if toggles.FORM_LINK_WORKFLOW.enabled(req.user.username):
+            modules = filter(lambda m: m.case_type == module.case_type, app.get_modules())
```

Comment:
I'd be interested to sync when you've nailed down the inclusion criteria so I have a better understanding of the bigger picture